src/engine/SCons/Tool/packaging/__init__.py

```diff
@@ -122,8 +121,17 @@ def Package(env, target=None, source=None, **kw):
     # load the needed packagers.
     def load_packager(type):
         try:
-            file,path,desc=imp.find_module(type, __path__)
-            return imp.load_module(type, file, path, desc)
+            if PY2:
```

Comment:
I think I have an alternate solution for this location... we just want a relative import as we're already in `SCons/Tool/Packaging`. For that, the following seems to work (for 2.7 as well)
```
        try:
            return importlib.import_module("." + type, __name__)
```